### DISCUSSION TITLE
How to pass today's date dynamically in rollup where clause

### DISCUSSION THREAD

**Discussion by Phanishankarp** [jamessimone/apex-rollup#495] [General]
What is the correct way to pass today's date dynamically 
<img width="344" alt="image" src="https://github.com/jamessimone/apex-rollup/assets/26519788/56700e4d-6995-4597-b673-0ae4ac1f7e35">

**Comment by jamessimone** [comment]
You would use a [SOQL date literal](https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_soql_select_dateformats.htm)

**Reply by Phanishankarp** [reply]
Thank you for your response. I am getting below error if I use **TODAY**
An error occurred: Where clause entered incorrectly: SBQQ__TerminatedDate__c != null and SBQQ__TerminatedDate__c <TODAY

<img width="941" alt="image" src="https://github.com/jamessimone/apex-rollup/assets/26519788/727812b9-9ab2-4b0b-804e-74a0b749a4b7">

**Reply by jamessimone** [reply]
@Phanishankarp you should be ok if you put a space between < and TODAY

**Reply by Phanishankarp** [reply]
Thanks a lot @jamessimone. It resolved the issue.
